src/app/frontend/deploy/deploy_controller.js

```diff
@@ -23,14 +23,12 @@ import {
  */
 export default class DeployController {
   /**
-   * @param {!angular.$resource} $resource
-   * @param {!angular.$log} $log
    * @param {!ui.router.$state} $state
    * @param {!backendApi.NamespaceList} namespaces
    * @param {!backendApi.Protocols} protocols
    * @ngInject
    */
-  constructor($resource, $log, $state, namespaces, protocols) {
+  constructor($state, namespaces, protocols) {
```

Comment:
Could you also clean up in test initialization of controllers you've changed? There are some unnecessary services injected.